cygnus-ngsi-ld/src/test/java/com/telefonica/iot/cygnus/sinks/NGSICKANSinkTest.java

```diff
@@ -0,0 +1,1042 @@
+        String backendMaxConnsPerRoute = null; // default
+        String batchSize = null; // default
+        String batchTime = null; // default
+        String batchTTL = null; // default
+        String dataModel = null; // default
+        String enableEncoding = null; // default
+        String enableGrouping = null; // default
+        String enableLowercase = null; // default
+        String host = null; // default
+        String port = null; // default
+        String ssl = null; // default
+        String viewer = null; // default
+        String entityId = ""; //defalut
+        NGSICKANSink sink = new NGSICKANSink();
+        sink.configure(createContext(apiKey, attrPersistence, backendMaxConns, backendMaxConnsPerRoute, batchSize,
+                batchTime, batchTTL, dataModel, enableEncoding, enableGrouping, enableLowercase, host, port, ssl,
+                viewer));
+        String entity = "veryLooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo"
+                + "ooooogEntity";
+        
+        try {
+            sink.buildResName(entity, entityId);
+            System.out.println(getTestTraceHead("[NGSICKANSink.buildResName]")
+                    + "- FAIL - A resource name length greater than 100 characters has not been detected");
+            assertTrue(false);
+        } catch (Exception e) {
+            assertTrue(true);
+            System.out.println(getTestTraceHead("[NGSICKANSink.buildResName]")
+                    + "-  OK  - A resource name length greater than 100 characters has been detected");
+        } // try catch
+    } // testBuildResourceNameLength
+    
+    private Context createContext(String apiKey, String attrPersistence, String backendMaxConns,
+            String backendMaxConnsPerRoute, String batchSize, String batchTime, String batchTTL, String dataModel,
+            String enableEncoding, String enableGrouping, String enableLowercase, String host, String port,
+            String ssl, String viewer) {
+        Context context = new Context();
+        context.put("api_key", apiKey);
+        context.put("attr_persistence", attrPersistence);
+        context.put("backend.max_conns", backendMaxConns);
+        context.put("backend.max_conns_per_route", backendMaxConnsPerRoute);
+        context.put("batch_size", batchSize);
+        context.put("batch_time", batchTime);
+        context.put("batch_ttl", batchTTL);
+        context.put("ckan_host", host);
+        context.put("ckan_port", port);
+        context.put("ckan_viewer", viewer);
+        context.put("data_model", dataModel);
+        context.put("enable_encoding", enableEncoding);
+        context.put("enable_grouping", enableGrouping);
+        context.put("enable_lowercase", enableLowercase);
+        context.put("ssl", ssl);
+        return context;
+    } // createContext
+
+   /* private NotifyContextRequestLD.ContextElement createContextElement() {
```

Comment:
Large blocks of code commented (I have spotted this one, but maybe there are more)

Some alternatives:

1. Fix the test and re-enable it (preferred one)
2. Remove the test if that is old stuff no longer applicable to the new sink
3. Add a FIXME mark about this test is still valuable but it will be fixed in a next PR

Reply:
Sorry, I forgot to perforn the cleaned up of the tests file before to commit. Right now this is fixed